src/pathfinding_service/database.py

```diff
@@ -0,0 +1,79 @@
+import os
+import sqlite3
+from typing import Optional
+
+import structlog
+
+from pathfinding_service.model import IOU
+from raiden_libs.types import Address
+
+log = structlog.get_logger(__name__)
+SCHEMA_FILENAME = os.path.join(
+    os.path.dirname(os.path.realpath(__file__)),
+    'schema.sql',
+)
+
+
+def convert_hex(raw: bytes) -> int:
+    return int(raw, 16)
+
+
+sqlite3.register_converter('HEX_INT', convert_hex)
+
+
+class PFSDatabase:
+    """ Store data that needs to persist between PFS restarts """
+
+    def __init__(self, filename: str, pfs_address: Address):
+        log.info('Opening database at ' + filename)
+        self.conn = sqlite3.connect(
+            filename,
+            detect_types=sqlite3.PARSE_DECLTYPES,
```

Comment:
Should we disable implicit tx handling here as well?

Reply:
Yes, good point!